Markdown linting and Langauge fixes (#123)
eficode-academy/kubernetes-katas: accessing-your-application.md

<details>
<summary>:bulb: How does this port-forward work?</summary>

> Port forwarding is a network address translation that redirects Internet packets from one IP address
> to another with a specified port number to another `IP:PORT` set.

Reviewer: ```suggestion
Port forwarding listens for incoming Internet packets on a specific port on one IP address and forwards them to another, predefined `IP:PORT` set.
```

In Kubernetes `port-forward` creates a tunnel between your local machine and the Kubernetes cluster on
the specified `IP:PORT` pairs to establish a connection to the cluster.

> `kubectl port-forward` allows you to forward not only pods but also services, deployments, and others.

Reviewer: ```suggestion
The `port-forward` command in Kuberentes forwards incoming traffic to the machine, the command is executed on, to the specified `IP:PORT` pairs inside the Kubernetes cluster. This effectively lets traffic from outside the cluster reach applications running inside the cluster.
`kubectl port-forward` allows you to forward traffic to pods, services, deployments, and others.
```

> More information can be found from [Use Port Forwarding to Access Applications in a Cluster](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)

Reviewer: ```suggestion
More information can be found in the Kubernetes docs at [Use Port Forwarding to Access Applications in a Cluster](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
```

</details>

> - Check that the pod is running with the `kubectl get pods` command.

Reviewer: ```suggestion
- Use the `kubectl get pods` command to verify that the pod is running.
```

> - Expose the frontend with port-forward

Reviewer: ```suggestion
- Expose the frontend with `port-forward`
```

Port forwarding can be achieved with:

`kubectl port-forward --address 0.0.0.0 frontend 8080:5000`

> > :bulb: We add the `--address 0.0.0.0` option to the port-forward command to make it accept
> > commands coming from remote machines, like your laptop! `0.0.0.0` Means any address, so you

Reviewer: ```suggestion
> :bulb: The `--address 0.0.0.0` option enables machines _other_ than the one running the `port-forward` command to use the port forwarding.
```

> probably don't want to do this on your machine unless you want to expose something

> > to the internet.

Reviewer: > This is _not safe_, but necessary if you are running the `port-forward` in an online training infrastructure
> and want to test the port forwarding from the browser on your laptop. You do not usually want to use this option.

### Clean up

> - Stop the port-forward with the `Ctrl+C` command.

Reviewer: ```suggestion
- Stop the port-forward by pressing `Ctrl+C`.
```
